src/intl/en.json

```diff
@@ -155,5 +155,113 @@
   "language-swedish": "Swedish",
   "language-turkish": "Turkish",
   "language-chinese": "Chinese Simplified",
-  "website-last-updated": "Website last updated"
+  "website-last-updated": "Website last updated",
+  "page-what-is-ethereum-meta-title": "What is Ethereum?",
+  "page-what-is-ethereum-meta-description": "Learn about Ethereum, what it does and how to try it for yourself.",
+  "page-what-is-ethereum-title": "What is Ethereum?",
+  "page-what-is-ethereum-desc": "The foundation for our digital future",
+  "page-what-is-ethereum-accessibility": "Ethereum is open to everyone.",
+  "page-what-is-ethereum-tools-needed": "All you need is a wallet to take part.",
+  "page-what-is-ethereum-alt-img-bazaar": "Illustration of a person peering into a bazaar, meant to represent Ethereum",
+  "page-what-is-ethereum-in-depth-description": "Ethereum is open access to digital money and data-friendly services for everyone – no matter your background or location. It's a community-built technology behind the cryptocurrency Ether (ETH) and thousands of applications you can use today.",
+  "page-what-is-ethereum-alt-img-social": "An illustration of characters in a social space dedicated to Ethereum with a large ETH logo",
+  "page-what-is-ethereum-welcome": "Welcome to Ethereum",
+  "page-what-is-ethereum-welcome-2": "We hope you stay.",
+  "page-what-is-ethereum-101": "Ethereum 101",
+  "page-what-is-ethereum-101-desc": "Ethereum is a technology that lets you send cryptocurrency to anyone for a small fee. It also powers applications that everyone can use and no one can take down.",
+  "page-what-is-ethereum-101-strong": "It's ",
```

Comment:
@ph5500 FYI - we shouldn't start or end these phrases with a space. Instead, we should add a space in the code component when necessary.